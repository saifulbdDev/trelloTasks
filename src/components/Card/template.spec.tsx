import React from "react";
import { shallow } from "enzyme";
import { Card } from "./";

describe("Card", () => {
  const props = { id: 1, text: "abc", value: "", category: "abc" };
  const card = shallow(<Card {...props} />);
  it("renders properly", () => {
    expect(card).toMatchSnapshot();
  });
});
