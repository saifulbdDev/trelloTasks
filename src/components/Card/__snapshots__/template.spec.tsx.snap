// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders properly 1`] = `ShallowWrapper {}`;
